packages/messenger-widget/src/hooks/server-side/ServerSideConnector.ts

```diff
@@ -89,23 +93,39 @@ export abstract class ServerSideConnector extends JwtInterceptor {
     }
 
     private async reAuth() {
-        //TODO check if we need alias subdomain
-        const url = `${this.baseUrl}/auth/${normalizeEnsName(this.ensName)}`;
+        try {
+            //TODO check if we need alias subdomain
+            const url = `${this.baseUrl}/auth/${normalizeEnsName(
+                this.ensName,
+            )}`;
 
-        const { data: challenge } = await axios.get(url);
+            const { data: challenge } = await axios.get(url);
 
-        const signature = await sign(
-            this.profileKeys.signingKeyPair.privateKey,
-            challenge,
-        );
+            const signature = await sign(
+                this.profileKeys.signingKeyPair.privateKey,
+                challenge,
+            );
 
-        //Todo move to lib
-        const { data: newToken } = await axios.post(url, {
-            signature,
-            challenge,
-        });
+            //Todo move to lib
+            const {
+                data: newToken,
+                status,
+                data: error,
+            } = await axios.post(url, {
+                signature,
+                challenge,
+            });
 
-        return newToken;
+            return newToken;
```

Comment:
please add some comments why this step might occur

Reply:
added